.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Could you explain why this folder is defined on `.gitignore` file and let this be not under Git version control?

Which tool generate this folder?

Reply:
It's an IDE folder generated by JetBrains PHP Storm. Let me remove it from .gitignore :)